fontbe/src/glyphs.rs

```diff
@@ -386,6 +386,10 @@ impl TryFrom<&ir::Glyph> for CheckedGlyph {
             })
             .collect();
         if path_els.len() > 1 {
+            warn!(
+                "{} has inconsistent path elements: {path_els:?}",
+                glyph.name
```

Comment:
can you not put also `glyph.name` inside the string-formatting braces, like you did for `path_els`?

```suggestion
                "{glyph.name} has inconsistent path elements: {path_els:?}",
```